wrappers/flutter/lib/pigeon.dart

```diff
@@ -248,12 +217,12 @@ class FitatuBarcodeScannerHostApi {
     }
   }
 
-  Future<void> setTorchEnabled(bool arg_isEnabled) async {
+  Future<void> release() async {
     final BasicMessageChannel<Object?> channel = BasicMessageChannel<Object?>(
-        'dev.flutter.pigeon.FitatuBarcodeScannerHostApi.setTorchEnabled', codec,
+        'dev.flutter.pigeon.FitatuBarcodeScannerHostApi.release', codec,
```

Comment:
setTorchEnabeld -> release? o co tu chodzi ? :D

Reply:
To Pigeon API do komunikacji

Reply:
chodziło mi skąd zmiana channelu, ale ten komentarz chyba zbędny bo tam chyba tylko kolejność się zmieniła i github jakoś dziwnie tak pokazuje

Reply:
Nazwy się zmieniły

Reply:
Bardziej wygląda jak zmiana formatowania, używałeś już wcześniej obu tych nazw 
![image](https://github.com/SymetriaSpJ/zxing-cpp/assets/19805864/d26c3664-9eb4-4002-95e2-5c08f3c0c569)
coś się zmieniło w komunikacji? (teraz na inny chanel wpadają wiadomości czy coś  / czemu?)

Reply:
To jest wygenerowany kod. Patrz plik pigeon/api.dart